Tests/ViewInspectorTests/SwiftUI/CustomViewModifierTests.swift

```diff
@@ -177,6 +177,144 @@ final class ModifiedContentTests: XCTestCase {
+            .modifier(AccessibilityEnabledEnvironmentModifier())
+            .modifier(AccessibilityEnabledEnvironmentModifier())
+            .modifier(AccessibilityEnabledEnvironmentModifier())
+            .modifier(AccessibilityEnabledEnvironmentModifier())
+            .modifier(AccessibilityEnabledEnvironmentModifier())
+            .modifier(AccessibilityEnabledEnvironmentModifier())
+            .modifier(AccessibilityEnabledEnvironmentModifier())
+            .modifier(AccessibilityEnabledEnvironmentModifier())
+            .modifier(AccessibilityEnabledEnvironmentModifier())
+
+        let sut = try view.inspect()
+        XCTAssertEqual(sut.content.medium.environmentModifiers.count, 15)
+        XCTAssertEqual(sut.content.medium.viewModifiers.count, 15)
+    }
+
+    func testMultipleEnvironmentObjects() throws {
+        let view = Text("str")
+            .environmentObject(ExternalState())
+            .environmentObject(ExternalState())
+            .environmentObject(ExternalState())
+            .environmentObject(ExternalState())
+            .environmentObject(ExternalState())
+            .environmentObject(ExternalState())
+            .environmentObject(ExternalState())
+            .environmentObject(ExternalState())
+            .environmentObject(ExternalState())
+            .environmentObject(ExternalState())
+            .environmentObject(ExternalState())
+            .environmentObject(ExternalState())
+            .environmentObject(ExternalState())
+            .environmentObject(ExternalState())
+            .environmentObject(ExternalState())
+
+        let sut = try view.inspect()
+        XCTAssertEqual(sut.content.medium.environmentObjects.count, 15)
+        XCTAssertEqual(sut.content.medium.viewModifiers.count, 0)
+    }
+
+    func testMultipleCustomEnvironmentObjectModifiers() throws {
+        let view = Text("str")
+            .modifier(ExternalStateEnvironmentObjectModifier())
+            .modifier(ExternalStateEnvironmentObjectModifier())
+            .modifier(ExternalStateEnvironmentObjectModifier())
+            .modifier(ExternalStateEnvironmentObjectModifier())
+            .modifier(ExternalStateEnvironmentObjectModifier())
+            .modifier(ExternalStateEnvironmentObjectModifier())
+            .modifier(ExternalStateEnvironmentObjectModifier())
+            .modifier(ExternalStateEnvironmentObjectModifier())
+            .modifier(ExternalStateEnvironmentObjectModifier())
+            .modifier(ExternalStateEnvironmentObjectModifier())
+            .modifier(ExternalStateEnvironmentObjectModifier())
+            .modifier(ExternalStateEnvironmentObjectModifier())
+            .modifier(ExternalStateEnvironmentObjectModifier())
+            .modifier(ExternalStateEnvironmentObjectModifier())
+            .modifier(ExternalStateEnvironmentObjectModifier())
+
+        let sut = try view.inspect()
+        XCTAssertEqual(sut.content.medium.environmentObjects.count, 15)
```

Comment:
This test fails this assertion in the state of f52f300f10df9a5980dcf839f0048a9dcccf6e72 (the first commit of this branch).

The `environmentObjects.count` returns **32767** items.

<img width="1105" alt="Screenshot 2023-12-13 at 10 10 10 PM" src="https://github.com/nalexn/ViewInspector/assets/2278022/b18a63f7-d66e-493d-9786-a8f13ee1ea8a">